flask_resources/resources.py

```diff
@@ -12,7 +12,7 @@
 
 from .deserializers import JSONDeserializer
 from .loaders import RequestLoader
-from .parsers import search_request_parser
+from .parsers import ArgsParser, search_request_parser
```

Comment:
minor:
```suggestion
from .parsers import ArgsParser
```

Reply:
Good, will remove it in #63 to not have to rebase...